src/course/views.py

```diff
@@ -134,7 +132,67 @@ def post(self):
                 current_user.courses.append(course)
                 models.db.session.commit()
                 author = models.User.query.filter_by(id=course.teacher_id).first()
-                print course.id
                 return "Redirect to:%s" % (url_for("view_course", courseID=course.id+1000))
         else:
             return "Registration Code Incorrect"
+
+class AddTAView(MethodView):
+
+    def get(self):
+        return
+
+    def post(self):
+        data = flask.request.get_json()
+        email = data.get('email')
+        courseId = data.get('courseID')
+        course = models.Course.query.filter_by(id=courseId).first()
+        if email:
+            user = models.User.query.filter(models.User.email.contains(email)).first()
+            if user and user.permissions:
+                    if user.permissions < 20:
+                        user.permissions = 20
+                    secondaryTeachers = course.secondaryTeachers
+                    userIdAndComma = str(user.id) + ","
```

Comment:
There is a better way to do this.

```python
secondaryTeachers = [t.strip for t in course.secondaryTeachers.split(",")]
secondaryTeachers.append(str(user.id))
course.secondaryTeachers = ", ".join(secondaryTeachers)
```

I dont think you need the .find, and this will prevent you from ending up with superflous trailing commas.